README.md

```diff
@@ -77,6 +78,20 @@ Use `make list-install-targets` to retrieve all the available install targets.
 > CI. Some of this setup runs as the `root` user and **could break your developer's
 > system**. See [Developer Mode](#developer-mode).
 
+### Controlling the CI
+
+#### Github Actions
+
+Kata uses Github Actions in the [kata-containers](https://github.com/kata-containers/kata-containers) repos.
+There are no trigger phrases to invoke these actions as they are triggered by creating a pull request.
+
+#### Jenkins
+
+The Jenkins configuration and most documentation is kept in the [CI repository](https://github.com/kata-containers/ci).
+Jenkins is setup to trigger a CI run on all the slaves/nodes when a "/test" comment is added to a pull request. However,
+there are some specific comments that are defined for specific CI slaves/nodes which are defined in the Jenkins
+`config.xml` files in the `<triggerPhase>` XML element in the [CI repository](https://github.com/kata-containers/ci).
```

Comment:
What about also adding a link to https://github.com/kata-containers/community/wiki/Controlling-the-CI ?

Reply:
@fidencio for this PR I do not want to include a link to that wiki as it is not updated and has a lot of information that is working with kata 2.x, I will create another PR after I update the wiki and put the current trigger phrases for kata 2.x ci jobs

Reply:
@fidencio for this PR I do not want to include a link to that wiki as it is not updated and has a lot of information that is working with kata 2.x, I will create another PR after I update the wiki and put the current trigger phrases for kata 2.x ci jobs

Reply:
Okay, makes sense.